src/marqo/upgrades/v20_v21.py

```diff
@@ -0,0 +1,90 @@
+import os
+
+from marqo import logging, version
+from marqo.api import exceptions as api_exceptions
+from marqo.core.index_management.index_management import IndexManagement, _MarqoConfig
+from marqo.upgrades.upgrade import Upgrade
+from marqo.vespa.models import VespaDocument
+from marqo.vespa.vespa_client import VespaClient
+
+logger = logging.get_logger(__name__)
+
+
+class v20v21(Upgrade):
```

Comment:
Can the naming be better like `class v2_0_0_v2_1_0`? This naming seems like it could conflict once we get to double digit major and minor versions.

Reply:
How about now? Python class names should be PascalCase so changed to v2_v0_v2_v1 (V2V0V2V1 class name)